firebase-dynamic-links/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 
 # 22.1.0
-* [changed] Added deprecation annotations to the public API. See https://firebase.google.com/support/dynamic-links-faq for further context.
+* [deprecated] Added deprecation annotations to the public API. See https://firebase.google.com/support/dynamic-links-faq for further context.
```

Comment:
Rachel used a different text in the internal CL. Do we want to use the same text here for consistency?